Sources/BraveWallet/Crypto/Portfolio/AddCustomAssetView.swift

```diff
@@ -36,10 +56,36 @@ struct AddCustomAssetView: View {
     self.userAssetStore = userAssetStore
     self.networkSelectionStore = .init(mode: .formSelection, networkStore: networkStore)
   }
+  
+  private var assetTypes: [TokenType] = [.token, .nft]
+  
+  private var addButtonDisabled: Bool {
+    if tokenType == TokenType.nft.rawValue {
+      return nameInput.isEmpty || symbolInput.isEmpty || (networkSelectionStore.networkSelectionInForm?.coin != .sol && tokenId.isEmpty) || addressInput.isEmpty || networkSelectionStore.networkSelectionInForm == nil || (networkSelectionStore.networkSelectionInForm?.coin != .sol && !addressInput.isETHAddress)
+    } else {
+      return nameInput.isEmpty || symbolInput.isEmpty || decimalsInput.isEmpty || addressInput.isEmpty || networkSelectionStore.networkSelectionInForm == nil || (networkSelectionStore.networkSelectionInForm?.coin != .sol && !addressInput.isETHAddress)
+    }
+  }
+  
+  private var showTokenID: Bool {
+    if let customAssetNetwork = networkSelectionStore.networkSelectionInForm, customAssetNetwork.coin == .sol {
+      return false
+    }
+    return true
+  }
 
   var body: some View {
     NavigationView {
       Form {
+        Section {
+        } header: {
+          Picker("What kind of asset do you want to add", selection: $tokenType) {
```

Comment:
localize

Reply:
removed this label since we dont need it. [299d530](https://github.com/brave/brave-ios/pull/6360/commits/299d530805abcc2e35a9cce53897e5f42c774c62)